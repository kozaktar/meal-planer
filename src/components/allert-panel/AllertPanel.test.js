import React from 'react'
import {shallow} from 'enzyme'
import AllertPanel from './AllertPanel.component'

describe('AllertPanel test', () => {

    it('AllertPanel is rendering without crashing', ()=>{
        expect(shallow(<AllertPanel/>)).toMatchSnapshot();
    })
})