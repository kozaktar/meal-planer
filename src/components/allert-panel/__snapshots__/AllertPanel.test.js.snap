// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllertPanel test AllertPanel is rendering without crashing 1`] = `
<WithStyles(ForwardRef(Snackbar))
  autoHideDuration={3000}
>
  <Alert />
</WithStyles(ForwardRef(Snackbar))>
`;
